openfisca_us/parameters/gov/states/in/tax/income/deductions/unemployment_compensation/agi_reduction.yaml

```diff
@@ -0,0 +1,21 @@
+description: Reduction to AGI for computation of unemployment compensation deduction.
+metadata:
+  period: year
+  unit: currency-USD
+  name: in_agi_reduction_unemployment_compensation_deduction
+  label: AGI reduction for calculation of unemployment compensation deduction in IN
+  breakdown:
+    - filing_status
+SINGLE:
+  2021-01-01: 12_000
+HEAD_OF_HOUSEHOLD:
+  2021-01-01: 12_000
+JOINT:
+  2021-01-01: 18_000
+WIDOW:
+  2021-01-01: 12_000
+SEPARATE:
+  2021-01-01: 12_000
```

Comment:
Does (3)(B) refer to separate filers?
>(B) Zero dollars ($0) if the individual:
(i) is married at the close of the taxable year, as determined under Section 7703 of the Internal Revenue Code;
(ii) does not file a joint return for the taxable year; and
(iii) does not live apart from the individual's spouse at all times during the taxable year.

Reply:
@MaxGhenis, a note in the instructions clarifies this: 

> If you were married but filing separately, and you lived with your spouse at any time during the year, enter -0- on line 3 of the worksheet. However, if you were MFS and lived apart from your spouse the entire year, enter $12,000 on line 3. 

My instinct is to leave the parameter as is ($12k) while adding a comment about this, but let me know if you think we should set to $0 by default for MFS or follow some other path.